services/director-v2/src/simcore_service_director_v2/modules/dynamic_sidecar/scheduler/task.py

```diff
@@ -321,28 +323,40 @@ async def _observing_single_service(service_name: str) -> None:
                 #   dy-sidecar, dy-proxy, or containers) -> it cannot be removed safely
                 # 4. service started, and failed on closing -> it cannot be removed safely
 
-                failed_while_saving_state_and_outputs = (
-                    scheduler_data.dynamic_sidecar.service_removal_state.can_save
-                    and scheduler_data.dynamic_sidecar.were_containers_created
-                )
-                if failed_while_saving_state_and_outputs:
+                if scheduler_data.dynamic_sidecar.skip_sidecar_monitor_and_removal:
                     # use-cases: 3, 4
                     # Since user data is important and must be saved, take no further
                     # action and wait for manual intervention from support.
-                    return
 
-                # use-cases: 1, 2
-                await cleanup_sidecar_stack_and_resources(
-                    dynamic_sidecar_settings, scheduler_data
-                )
+                    # After manual intervention service can now be removed
+                    # from tracking.
+                    if (
+                        # trigger every ~30 seconds to reduce pressure on
+                        # docker swarm engine API.
+                        # NOTE: do not change below order
+                        self._observation_counter % 6 == 0
+                        and await is_dynamic_sidecar_stack_missing(
+                            scheduler_data.node_uuid, dynamic_sidecar_settings
+                        )
+                    ):
+                        # if both proxy and sidecar ar missing at this point it
+                        # is safe to assume that user manually removed them from
+                        # Portainer after cleaning up.
```

Comment:
question: what will happen if this runs while a node is restarting, or missing from the swarm, where the sidecar will actually restart? won't we be in such a case? what will happen here?